src/Components/TableRepeater.php

```diff
@@ -42,6 +44,13 @@ public function emptyLabel(bool|string|Closure $label = null): static
         return $this;
     }
 
+    public function headersTextAlignment(string|Closure $alignment = 'left'): static
+    {
+        $this->headersTextAlignment = $alignment;
+
+        return $this;
+    }
```

Comment:
```php
public function headersAlignment(string|Closure $alignment = 'left'): static
{
    $this->headersAlignment = $alignment;

    return $this;
}
```

Reply:
are we going with alignHeaders or headersAlignment?

Reply:
Let's go with 'alignHeaders', I think that reads more fluently. Thanks.